avocado/plugins/run.py

```diff
@@ -85,6 +85,10 @@ def configure(self, parser):
                             default='off', help='Keep job temporary files '
                             '(useful for avocado debugging). Defaults to off.')
 
+        parser.add_argument('--force-execution', choices=('on', 'off'),
```

Comment:
I'd not like to use 'skip' here because we have the `SKIP` status and it might confuse people. I think `ignore-missing-references` makes a lot of sense though.